.travis.yml

```diff
@@ -40,7 +40,7 @@ _lint_job: &lint_job
     - bash .github/lint-disallowed-functions-in-library.sh
     - bash .github/lint-commit-message.sh
     - bash .github/lint-filename.sh
-    - golangci-lint run ./...
+    - golangci-lint run --build-tags quic ./...
```

Comment:
Mind removing this line? We use [this](https://github.com/pion/.goassets) repo to keep all our configs in sync. Maybe we can just use `TEST_EXTRA_ARGS` here, or we need to add a new `GOLANGCI_LINT_EXRA_ARGS`

But this can be a follow up :)